lib/test/unit/ui/console/testrunner.rb

```diff
@@ -394,6 +395,12 @@ def output_statistics(elapsed_time)
             change_output_level(IMPORTANT_FAULTS_ONLY) do
               output("Finished in #{elapsed_time} seconds.")
             end
+            if @options[:report_slow_tests]
+              @test_statistics.sort_by {|slow_test| -slow_test[:elapsed_time]}
+                              .first(N_REPORT_SLOW_TESTS)
+                              .each do |slow_test|
```

Comment:
Could you use better variable name?

```suggestion
              @test_statistics.sort_by {|statistic| -statistic[:elapsed_time]}
                              .first(N_REPORT_SLOW_TESTS)
                              .each do |slow_statistic|
```

Reply:
Thanks for your suggestion. There are better variable name!